src/scvi/external/__init__.py

```diff
@@ -4,6 +4,7 @@
 from .gimvi import GIMVI
 from .methylvi import METHYLANVI, METHYLVI
 from .mrvi import MRVI
+from .nichevi import nicheSCVI
```

Comment:
At least first letter large for classes. Just call the class NICHEVI - will be more user friendly.

Reply:
Duplicate.